src/analysis/souffle/taint.dl

```diff
@@ -28,6 +28,10 @@
 // paths. For the time being, we will use a `symbol` to represent access paths.
 // However, in the long term, it would make sense to define an ADT for this.
 //
+
```

Comment:
@markww it might be worthwhile sending this out as separate PR and merge it in, while you work on the analysis.